soroban-env-host/src/host/ledger_info_helper.rs

```diff
@@ -16,6 +16,44 @@ impl Host {
                 self.with_ledger_info(|li: &LedgerInfo| Ok(li.min_persistent_entry_expiration))?
             }
         };
-        Ok(ledger_seq.saturating_add(min_expiration))
+        Ok(ledger_seq.saturating_add(min_expiration.saturating_sub(1)))
```

Comment:
This is a bit arbitrary, but the idea is that the entry should live for `min_expiration` ledgers exactly. Since it's introduced in `ledger_seq` ledger, then its expiration ledger is `ledger_seq + min_expiration - 1`. The entry is still alive on its expiration ledger.